Formula/node.rb

```diff
@@ -56,30 +56,11 @@ def install
 
     if build.with? "npm"
       resource("npm").stage buildpath/"npm_install"
-
-      # make sure npm can find node
-      ENV.prepend_path "PATH", bin
```

Comment:
Oh, I misunderstood you, apologies. I was thinking in terms of the npm arguments likely having to be tweaked on the `brew` side, which could probably be flexible depending on which `node` was being used.

But yes, you are correct on the clarification. Either that or we permanently stuff `npm` inside the keg for the older `node`s but set the `prefix` in `npmrc` to something like `#{var}/node6modules` so user-installed stuff isn't mushed across reinstalls/upgrades/etc 😖.